packages/utils/src/scripts/gecko/custom-tokens.json

```diff
@@ -13,5 +13,25 @@
     "id": "danchortoken",
     "symbol": "usn",
     "name": "USN"
+  },
+  {
+    "id": "chexchexchex",
+    "symbol": "chex",
+    "name": "Chintai"
+  },
+  {
+    "id": "eoshashcoins",
+    "symbol": "hash",
+    "name": "Hash"
+  },
+  {
+    "id": "eoshashcoins",
```

Comment:
the "id" fields should be unique otherwise the component creating and matching would have issues.